cmd/build.go

```diff
@@ -145,3 +154,16 @@ func hasMain(dir string) bool {
 	}
 	return foundMain
 }
+
+func openReport(_ *cobra.Command, args []string) {
+	if len(args) != 1 {
+		exitWithError("ID required")
+	}
+
+	report, err := tool.Build().(reco.BuildReporter).Report(args[0])
+	if err != nil {
+		exitWithError(err)
+	}
+	logger.Std.Printf("Build Report: %s", report)
+
+}
```

Comment:
Excess newline at end of function. If anything, I'd move the spare newline to before the logger.